rules/validator.go

```diff
@@ -125,6 +125,11 @@ func (v *validator) validateRollupRules(rrv map[string]*RollupRuleView) error {
 		}
 
 		for _, target := range view.Targets {
+			// Validate that rollup target name is valid
+			if err := validateChars(target.Name, v.opts.RollupTargetNameInvalidChars()); err != nil {
```

Comment:
nit: assign `invalidChars := v.opts. RollupTargetNameInvalidChars` outside the for loop and reuse it here? You could it just cache it inside the validator to avoid function calls completely.